server/ladokApi.js

```diff
@@ -27,10 +37,13 @@ async function getCourseRoundsData(courseCode, lang) {
       lastTuitionDate: round.sistaUndervisningsdatum.date,
       state: 'APPROVED',
       language: {
-        sv: round.undervisningssprak.name,
-        en: round.undervisningssprak.nameOther,
+        sv: round.undervisningssprak ? round.undervisningssprak.name : '',
+        en: round.undervisningssprak ? round.undervisningssprak.nameOther : '',
```

Comment:
English isn't always "the other" language? If `lang` is "en" then Swedish is the "other" language.

Reply:
Good catch! I'll fix it.